samples/net/sockets/websocket_client/README.rst

```diff
@@ -1,7 +1,8 @@
-.. _sockets-websocket-client-sample:
+.. zephyr:code-sample:: sockets-websocket-client
+   :name: Socket WebSocket Client
```

Comment:
I know you just copied the existing name, but I think it is now good time to change this to "WebSocket Client", the extra "Socket" does not give much information here and just sounds funny.

Reply:
ah! yeah I was thinking that all these "Socket... " were extraneous. Are we good to remove "Socket...." everywhere you think?  Even things like making "Socket echo client" -->  "Echo client"? I'd say yes?

Reply:
Yes, I agree, the word "socket" in many of these samples is quite pointless.

Reply:
done